src/EditorFeatures/CSharp/MisplacedUsings/MisplacedUsingsCodeFixProvider.UsingsSorter.cs

```diff
@@ -343,8 +343,8 @@ private void InsertBlankLinesBetweenSubGroups(List<UsingDirectiveSyntax> usingsL
                 var previousUsing = usingsList[0];
                 var root = GetNamespaceRoot(previousUsing.Name);
 
-                // We prime with the first using's Namespace root and the last using will always get a blank line.
-                for (var i = 1; i < usingsList.Count - 1; i++)
+                // We have already processed the first using's Namespace root.
+                for (var i = 1; i < usingsList.Count; i++)
```

Comment:
💭 Why did this change from `usingsList.Count - 1` to `usingsList.Count`?

Reply:
I was initially wrong in thinking I didn't have to look at the last node. The blank line is applied to the previous node, so a gap could be required.

Reply:
Can you make sure we have a test covering this case (i.e. would fail if this was changed back to `Count - 1`)?